code/modules/mob/language/generic.dm

```diff
@@ -26,7 +26,9 @@
 	whisper_verb = "whispers"
 	key = "0"
 	flags = RESTRICTED
-	syllables = list("vol", "zum", "coo","zoo","bi","do","ooz","ite","og","re","si","ite","ish","ar","at","on","ee","east","ma","da", "rim")
+	syllables = list(
+"vol", "zum", "coo","zoo","bi","do","ooz","ite","og","re","si","ite","ish",
+"ar","at","on","ee","east","ma","da", "rim")
```

Comment:
Why was this changed?

Reply:
Consistent with every other language this way.